src/icons/fileIcons.ts

```diff
@@ -243,7 +247,17 @@ export const fileIcons: FileIcons = {
       fileNames: ['astro.config.js', 'astro.config.mjs', 'astro.config.cjs'],
     },
     { name: 'pdf', fileExtensions: ['pdf'] },
-    { name: 'table', fileExtensions: ['xlsx', 'xls', 'csv', 'tsv', 'psv'] },
+    {
+      name: 'table',
+      fileExtensions: [
```

Comment:
There're some prettier issues, could you also take a look at these?

![image](https://user-images.githubusercontent.com/12248527/160403524-a87f820a-4cfe-4aa4-99c8-245dc31a9d14.png)

Reply:
my mistake, my prettier extension has been acting weird recently, just fixed them